kinto_client/utils.py

```diff
@@ -3,12 +3,19 @@
 from unidecode import unidecode
 import six
 
+REGEXP = re.compile(r'^[a-zA-Z0-9][a-zA-Z0-9_-]*$')
+
 
 def slugify(value):
     """Normalizes string, converts to lowercase, removes non-alpha characters
     and converts spaces to hyphens.
     """
-    value = unidecode(six.text_type(value))
+    value = six.text_type(value)
+    # Do not lowercase already valid value.
```

Comment:
All the value are existing :) I mean doesn't alterate values that are valid ids.